.github/workflows/build.yaml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on:
+  push:
+    branches:
+      - feature/workflows
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: /home/runner/work/open-feature/golang-sdk
+      GOBIN: /home/runner/work/open-feature/golang-sdk/bin
```

Comment:
I'm not sure how consistent the runner path is. It may be worth considering doing it the same way OTel does. You can see their workflow [here](https://github.com/open-telemetry/opentelemetry-go/blob/main/.github/workflows/ci.yml#L22).

Reply:
Done, thanks for the tip